cpp/src/binaryop/binaryop.cpp

```diff
@@ -46,11 +62,6 @@ namespace jit {
   std::string{std::getenv("CONDA_PREFIX")} + "/include/libcudf"
 #endif
```

Comment:
@jlowe can we remove this def to fix https://github.com/rapidsai/cudf/issues/3722?

Reply:
Nice catch, done. The bug was already fixed when the `libcudf_include_dir` definition was removed, as this is now just a #define that never gets referenced.